testproject/tests/test_second_step_authentication.py

```diff
@@ -78,28 +111,28 @@ def test_wrong_second_step_verification_with_ephemeral_token(
 ):
     client = APIClient()
     first_step = login(active_user_with_email_otp)
-    secret = active_user_with_email_otp.mfa_methods.first().secret
+    handler = get_mfa_handler(mfa_method=active_user_with_email_otp.mfa_methods.first())
     response = client.post(
-        path="/auth/login/code/",
+        path=PATH_AUTH_JWT_LOGIN_CODE,
         data={
             "ephemeral_token": first_step.data.get("ephemeral_token") + "wrong",
-            "code": create_otp_code(secret),
+            "code": handler.create_code(),
         },
         format="json",
     )
-    assert response.status_code == 400
+    assert response.status_code == 401
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too